workflows/methylation/methylation-preprocess.wdl

```diff
@@ -0,0 +1,98 @@
+version 1.1
+
+workflow methylation_preprocess {
```

Comment:
To prevent code duplication, I think this whole workflow should be axed. What is it doing that isn't handled by the task itself?

Reply:
Originally there was no `methylation-standard.wdl` and I didn't want a task entrypoint as a preprocessing step. Now, I'm not sure exactly how this will be run in production. It is a good question though. How do we handle "workflows" that only have a single step? Do we write a workflow or invoke the `task` directly?

Reply:
IMO we just use the task as the entrypoint. It's easy enough to run tasks using miniwdl. Granted, I don't think that's possible in cromwell? But we've already upgraded to 1.1 which isn't cromwell compatible, so not really a good reason to keep a workflow around.

My main concern with a "wrapper workflow" is that it should have dang-near identical meta entries, which is a source for discrepancies and maintenance headaches.